contracts/bonding/BondingVotes.sol

```diff
@@ -148,6 +148,10 @@ contract BondingVotes is ManagerProxyTarget, IBondingVotes {
 
     /**
      * @notice Returns the current amount of votes that `_account` has.
+     *
+     * The value returned by this function can be equivalently calculated using {BondingManager} functions as follows:
```

Comment:
I don't think this is true because if `_account` is a delegator delegated to a non-registered transcoder then this function will return 0 instead of `BondingManager.pendingStake(_account, 0)` as of the snapshot round.

Reply:
Ah, you're right. I thought it would be a simple adaptation from that simpler fix I mentioned (`bondedAmount == 0`). I'll see if I can represent the actual logic simply enough (maybe using english lol), otherwise will just revert this doc change.

Reply:
Fixed this on the same commit of `getBondingStateAt` rename.